src/components/plan/map/MapSection.tsx

```
@@ -0,0 +1,208 @@
+import { Map, CustomOverlayMap } from "react-kakao-maps-sdk";
```

Biome (2.1.2)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)